enterprise/replicasync/replicasync.go

```diff
@@ -274,7 +275,19 @@ func (m *Manager) syncReplicas(ctx context.Context) error {
 		wg.Add(1)
 		go func(peer database.Replica) {
 			defer wg.Done()
-			req, err := http.NewRequestWithContext(ctx, http.MethodGet, peer.RelayAddress, nil)
+			ra, err := url.Parse(peer.RelayAddress)
+			if err != nil {
+				m.logger.Warn(ctx, "could not parse relay address",
+					slog.F("relay_address", peer.RelayAddress), slog.Error(err))
+				return
+			}
+			target, err := ra.Parse("/derp")
```

Comment:
I think `/derp/latency-check` might be a bit better since it's meant for pinging, rather than just calling `/derp` incorrectly.

Reply:
That's a good idea.  Customers often log status codes, so 426 is going to get them wound up over nothing.